Sources/SourceKitLSP/SourceKitLSPServer.swift

```diff
@@ -2471,7 +2471,11 @@ fileprivate extension CheckedIndex {
   /// If the USR has an ambiguous definition, the most important role of this function is to deterministically return
   /// the same result every time.
   func primaryDefinitionOrDeclarationOccurrence(ofUSR usr: String) -> SymbolOccurrence? {
-    return definitionOrDeclarationOccurrences(ofUSR: usr).sorted().first
+    let result = definitionOrDeclarationOccurrences(ofUSR: usr).sorted().first
+    if result == nil {
+      logger.error("Failed to find definition of \(usr) in index")
```

Comment:
I assume you added this after something happened and this would have helped, out of interest... what was that?

Reply:
I think it was just trying to figure out why the test failed and I wanted to know whether LSP got to a USR and couldn’t find the USR, or if it never found the USR, or I wanted to check which USR it was actually looking for and if it was looking for the wrong USR. I don’t remember the details.